spec/models/waitlist_team_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe WaitlistTeam do
```

Comment:
I would also like to see the functionality of how waitlisting works when team gets deleted